### ISSUE TITLE
[IMPROVEMENT] Make state variable in App component clearer

### ISSUE THREAD

**Issue by brianbancroft** [data-mermaid/mermaid-dash#13]
The following state variable, `show`, is not clear in its naming. It needs to be changed to something that makes its use-case clear

```
  state = {
    show: true
  };

  toggle = () => {
    this.setState({ show: !this.state.show });
  };
```

**Comment by tienhoah** [comment]
https://github.com/data-mermaid/mermaid-dash/pull/18

**Comment by brianbancroft** [comment]
Closed by #18
